src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -515,16 +515,16 @@ public int Space_WithinSquares
             set { SetBooleanOption(CSharpFormattingOptions.SpaceWithinSquareBrackets, value); }
         }
 
-        public int Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
+        public string Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
```

Comment:
is this a breaking change?  @jasonmalinowski is this just for use by us, or do we expect 3rd parties to go through these entrypoints?